### PR TITLE
Fix unit test failure in memory usage for s390x

### PR DESCRIPTION
**Pull request by HarryLeeIBM**
Fix shaky functional test 02354_distributed_with_external_aggregation_memory_usage in s390x.

### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fixed functional test in 02354_distributed_with_external_aggregation_memory_usage in s390x.

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: tests/queries/0_stateless/02354_distributed_with_external_aggregation_memory_usage.sql [hunk 1]
```diff
@@ -23,6 +23,6 @@ select a, b, c, sum(a) as s
 from remote('127.0.0.{2,3}', currentDatabase(), t_2354_dist_with_external_aggr)
 group by a, b, c
 format Null
-settings max_memory_usage = '4Gi';
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/02354_distributed_with_external_aggregation_memory_usage.sql:26]: The comment is not updated.

**Comment by alexey-milovidov** [comment]
This is a functional test, no a unit test.

**Comment by alexey-milovidov** [comment]
We don't have support for s390x; therefore, the changelog category should be changed from
"Build/Testing/Packaging Improvement" to "Not for changelog".

@HarryLeeIBM 
You should add the support for s390x to our CI to speed up the processing of these pull requests.

**Comment by robot-clickhouse** [comment]
This is an automated comment for commit 95cb466e2b33d64296a13f7138f3cf4ea4b06a23 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/52073/95cb466e2b33d64296a13f7138f3cf4ea4b06a23/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by alexey-milovidov** [comment]
No updates in three weeks.

**Pull request closed without merging**